WcaOnRails/app/models/user.rb

```diff
@@ -886,7 +887,7 @@ def editable_fields_of_user(user)
         email preferred_events results_notifications_enabled
       )
       fields << { user_preferred_events_attributes: [:id, :event_id, :_destroy] }
-      if user.staff?
+      if user.staff? || user.trainee_delegate?
```

Comment:
Perhaps introduce a method like `staff_or_any_delegate?` that can be reused down the line

Reply:
(including a comment explaining that trainee delegates are not staff, to understand why querying `staff?` is not enough)